pkgs/development/compilers/flasm/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, unzip, bison, flex, gperf, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "flasm";
+  version = "1.64";
+
+  src = fetchurl {
+    url = "https://www.nowrap.de/download/flasm16src.zip";
```

Comment:
This download does not seem to have the full version number in it. Is there a better available?

Reply:
Unfortunately all zips on the official website seem to have only 16 in the name.